user_guide.adoc

```diff
@@ -380,6 +380,21 @@ enums:
     17: udp
 ----
 
+Alternatively, hexadecimal notation can also be used to define an enumeration:
```

Comment:
I was thinking that a new section of the document could be created for general syntax and a very brief overview of YAML and what it provides. This example I provided may be better suited there.